pkg/ui/userinterface.go

```diff
@@ -54,13 +56,26 @@ type consoleUi struct {
 	errorWriter        io.Writer
 	progressBarFactory func() ProgressBar
 	reader             *bufio.Reader
+	//nolint:containedctx // needed to preserve non-breaking change
+	context context.Context
```

Comment:
Issue: @j-luong let's remove the state here please. A possible solution is an internal structure to put this in stead of the UI instance